routing/routing_integration_tests/bicycle_turn_test.cpp

```diff
@@ -0,0 +1,56 @@
+#include "testing/testing.hpp"
+
+#include "routing/routing_integration_tests/routing_test_tools.hpp"
+
+#include "routing/route.hpp"
+
+using namespace routing;
+using namespace routing::turns;
+
+UNIT_TEST(RussiaMoscowSevTushinoParkBicycleWayTurnTest)
+{
+  TRouteResult const routeResult = integration::CalculateRoute(
+      integration::GetBicycleComponents(), MercatorBounds::FromLatLon(55.87467, 37.43658),
+      {0.0, 0.0}, MercatorBounds::FromLatLon(55.8719, 37.4464));
+
+  Route const & route = *routeResult.first;
+  IRouter::ResultCode const result = routeResult.second;
+  TEST_EQUAL(result, IRouter::NoError, ());
+
+  integration::TestTurnCount(route, 1);
+
+  integration::GetNthTurn(route, 0).TestValid().TestDirection(TurnDirection::TurnLeft);
+
+  integration::TestRouteLength(route, 752.);
+}
+
+UNIT_TEST(RussiaMoscowGerPanfilovtsev22BicycleWayTurnTest)
+{
+  TRouteResult const routeResult = integration::CalculateRoute(
```

Comment:
It's quite complex to debug such tests, because:
* it's hard to make integration_tests work on a developer machine, as a lot of maps needed
* it's hard to debug them, as I have no idea what happens at 55.85717 - 37.41052.

Why not to reuse TestMwmBuilder (if you need full mwm) and create a small but obvious road graph, with tests on it?